py/setup.cfg

```diff
@@ -4,8 +4,3 @@ exclude = .tox,docs/source/conf.py,*venv
 extend-ignore = E501, E203
 # This does nothing for now as E501 is ignored.
 max-line-length = 120
-
-[tool:pytest]
```

Comment:
Is this removal expected?

Reply:
@VietND96 Yes, because pytest settings are now maintained in `pyproject.toml`. It's not required to have it once again in `setup.cfg` file.